lib/rust/prelude/src/debug.rs

```diff
@@ -126,3 +126,75 @@ impl Drop for TraceCopies {
         }
     }
 }
+
+
+
+// ====================
```

Comment:
Should this be in the prelude, or is there a better place for it closer to the executor or other memory management functionality?

Reply:
It needs to be available from many places, including both IDE controllers and view. Also, the executor I know is more about handling asynchronous tasks, not memory management.

I cannot think about any other package.